src/ConstantSum/ConstantSumSolver.sol

```diff
@@ -81,13 +80,9 @@ contract ConstantSumSolver {
         bytes memory swapData;
 
         if (swapXIn) {
-            swapData = abi.encode(
-                0, 1, amountIn, state.amountOut, state.deltaLiquidity
-            );
+            swapData = abi.encode(0, 1, amountIn, state.amountOut);
         } else {
-            swapData = abi.encode(
-                1, 0, amountIn, state.amountOut, state.deltaLiquidity
-            );
+            swapData = abi.encode(1, 0, amountIn, state.amountOut);
```

Comment:
I notice `state.deltaLiquidity` is no longer being included in `swapData`. Either this is intentional and the function calls I commented on above can be removed, or it is unintentional and `deltaLiquidity` should be properly computed and included.

Reply:
It's intentional yes, the `deltaLiquidity` is now calculated by the strategies during a swap.